site/content/docs/2.0/utilities/text.md

```diff
@@ -69,6 +69,18 @@ Prevent long strings of text from breaking your components' layout by using `.te
 <p class="text-break">mmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm</p>
 {{< /example >}}
 
+## HYPHENATE WORD
```

Comment:
One minor cosmetic gripe: none of the other h2 headings in this part of the docs are all caps.

Reply:
Great catch! Thanks